radicle/src/cob/patch.rs

```diff
@@ -461,6 +481,20 @@ pub struct Merge {
     pub timestamp: Timestamp,
 }
 
+impl Merge {
+    pub fn node(&self) -> &NodeId {
+        &self.node
+    }
+
+    pub fn commit(&self) -> git::Oid {
+        self.commit
+    }
+
+    pub fn timestamp(&self) -> Timestamp {
+        self.timestamp
+    }
+}
+
```

Comment:
Can be removed

Reply:
You mean the entire impl block of the `Merge` struct?

Reply:
We need the Merge getters in different places

Reply:
The methods; Merge already has public fields

Reply:
Ah right, got it